src/test/ui/lifetime-errors/ex3-both-anon-regions.stderr

```diff
@@ -0,0 +1,10 @@
+error[E0623]: lifetime mismatch
+  --> $DIR/ex3-both-anon-regions.rs:12:12
+   |
+11 | fn foo(x: &mut Vec<&u8>, y: &u8) {
+   |                    ---      --- these references must have the same lifetime
```

Comment:
My worry is for example, a client that displays information for each span individually without the entire context of the error on hover. I feel that this is another case of conflicting requirements for the RLS and for `rustc` visual output.